docs/main.js

```diff
@@ -105,7 +105,7 @@
     var versionParam = version ? '?v=' + version : '';
     var code = event.target.nextElementSibling.textContent;
     var encoded = fixedEncodeURIComponent(code);
-    location.assign(location.origin + '/repl/' +
+    window.open(location.origin + '/repl/' +
       versionParam + '#;' + encoded);
```

Comment:
@buzzdecafe , sorry, just Barney Stinson raised up in my head :smile: with his rule:
![image](https://user-images.githubusercontent.com/8372070/27637627-12242f00-5c19-11e7-9714-65c5801415a3.png)

Just kidding :smile: apologies for the offtopic
I think the new tab is preferable because the user can, for example, try several approaches, and, IMHO, it's cooler to have a separate tab for each approach to have the possibility to compare them if is needed.